src/ts/index.js

```diff
@@ -144,6 +144,17 @@ const types = async (userTSConfig, opts) => {
     if (fs.existsSync(typesPath)) {
       fs.copySync(typesPath, fromRoot('dist', path.basename(typesPath)))
     }
+
+    await execa('copyup', [
+      'src/**/*.d.ts',
```

Comment:
Will not copy anything not in `src` - this pattern should be configurable

Reply:
do you have any specific uses for that ? i just followed our conventions of using `src` and `dist`

Reply:
i have been improving the aegir config step by step to be more flexible, we can add config for this later.
this should support most of our repos right now.